content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -227,6 +227,38 @@ To set up a team, repeat the instructions for individual users above to:
 
 Please read [Working with AWS credentials](#working-with-aws-credentials) to decide the right method for providing AWS IAM credentials for your O3DE project.
 
+## Running your O3DE project on Amazon EC2
+
+If you are running your project on Amazon EC2, you can utilize the [instance profile](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#ec2-instance-profile) to authenticate calls made to AWS. Using the IAM role from the Amazon EC2 instance profile lets you avoid configuring AWS credentials on the machine through less secure methods like supplying them as user data at deploy time or remotely accessing the machine to manually set up a profile.
+
+To use Amazon EC2 instance role credentials with your project:
+1. Create an Amazon EC2 [instance profile](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) through the [Amazon web console](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html#instance-profiles-manage-console) or AWS CLI. An instance profile is essentially a container for an IAM role that your Amazon EC2 instance can assume to make calls to AWS.
+1. Provide the associated IAM role any [permissions](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_manage-attach-detach.html) required to access the AWS resources your O3DE project needs.
+1. Attach the instance profile to the Amazon EC2 instance(s) running your O3DE project. You can attach it to a new instance [at launch time](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/LaunchingAndUsingInstances.html) or you can [attach it to a running instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html#attach-iam-role). 
+
+The AWS Core Gem also requires that the `AllowAWSMetadataCredentials` setting be set to `true` before it will query the [EC2 instance metadata service (IMDS)](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) for credentials. This default setting is to prevent unwanted calls to the Amazon EC2 IMDS endpoint when running your O3DE project locally or on non-Amazon EC2 compute.
```

Comment:
Just noticed the earlier debate about whether or not to mention this variable. My preference here is to include it inside a "note" callout. I'm not sure which setting overrides which, but I'm thinking of something like this:

{{< note >}}
If the environment variable `AWS_EC2_METADATA_DISABLED` is set, it will override the `AllowAWSMetadataCredentials` setting. For more information about this environment variable, refer to the [HttpRequestor Gem](/docs/user-guide/gems/reference/network/http-requestor/#turn-off-amazon-ec2-instance-metadata-service-calls).
{{< /note >}}

Note: I'm aware that the HttpRequestor Gem topic in the docs development branch does not have this new information, but I recommend going ahead and using this link. We'll be merging main into development very soon.